docs/howto/openstack/neutron/vpnaas.md

```diff
@@ -0,0 +1,727 @@
+    | Field  | Value |
+    +--------+-------+
+    | Status | DOWN  |
+    +--------+-------+
+    ```
+        
+    It looks like it is disabled --- or `DOWN`.
+        
+    > You will probably have to wait several seconds before seeing a 
+    status change. If you are impatient, try something like `watch -n 3 
+    "openstack vpn ipsec site connection show vpn-conn-to-kna1 -c Status"`. 
+    Hit CTRL+C as soon as you see the status change you expect.
+        
+    Now, get on the right side of the connection (region `kna1`), 
+    optionally look for the name of the VPN connection to the left (in our 
+    example, that would be `vpn-conn-to-fra1`), and check its status:
+        
+    ```bash
+    openstack vpn ipsec site connection show vpn-conn-to-fra1 -c Status
+    ```
+        
+    ```plain
+    +--------+-------+
+    | Field  | Value |
+    +--------+-------+
+    | Status | DOWN  |
+    +--------+-------+
+    ```
+    
+    It turns out that this direction of the connection is also disabled.
+
+## Enabling an inactive connection
+
+You can easily enable an inactive site-to-site VPN connection.
+
+=== "{{gui}}"
+    Currently, there is no way to enable an inactive connection from 
+    the {{gui}}. If you want to disable an inactive connection, please use 
+    the OpenStack CLI.
+=== "OpenStack CLI"
+    Make sure you hop on the side where you initially disabled the 
+    connection. According to the example scenario we described in the 
+    previous section, that would be the left side (region `fra1`), and the 
+    name of the disabled connection would be `vpn-conn-to-kna1`. Make sure 
+    the connection status is `DOWN`:
+        
+    ```bash
+    openstack vpn ipsec site connection show vpn-conn-to-kna1 -c Status
+    ```
+        
+    ```plain
+    +--------+-------+
+    | Field  | Value |
+    +--------+-------+
+    | Status | DOWN  |
+    +--------+-------+
+    ```
+        
+    Note that if you issued a similar command from the right side of 
+    the connection (region `kna1`), you would also get a `DOWN` status. 
+    Being on the left side, all you have to do to enable the inactive 
+    connection is type the following:
+        
+    ```bash
+    openstack vpn ipsec site connection set --enable vpn-conn-to-kna1
+    ```
+        
+    Check the connection status --- it should be `ACTIVE`:
+        
+    ```bash
+    openstack vpn ipsec site connection show vpn-conn-to-kna1 -c Status
+    ```
+        
+    ```plain
+    +--------+--------+
+    | Field  | Value  |
+    +--------+--------+
+    | Status | ACTIVE |
+    +--------+--------+
+    ```
+        
+    You get the same status by issuing a similar command from the right 
+    side.
+        
+    > Again, since you may have to wait several seconds before seeing 
+    the status change you expect, try something like this: `watch -n 3 
+    "openstack vpn ipsec site connection show vpn-conn-to-kna1 -c Status"`. 
+    Hit CTRL+C to stop watching.
```

Comment:
Here's how you get this properly formatted:

```markdown
    > Again, since you may have to wait several seconds before seeing 
    > the status change you expect, try something like this:
    > ```bash
    > watch -n 3 "openstack vpn ipsec site connection show vpn-conn-to-kna1 -c Status"
    > ```
    > Hit CTRL+C to stop watching.
```

Reply:
Updated, thanks :)

Reply:
You can probably remove `-n 3` as well, because it doesn't make that big of a difference if we refresh every 2 seconds (default) or every 3. :)